### DISCUSSION TITLE
Maintainers are not able to push to protected branches

### DISCUSSION THREAD

**Discussion by vmalyi** [community/community#36550] [Other Feature Feedback, Questions, & Ideas]
Hello everyone 👋

in our org, we have a repo with the protected `project` branche.

The issue we're facing is that if there is a PR attempting to merge `project/my-project-branch-name` to `main` and there is a conflict which needs to be fixed, only the repo admins are able to push conflict resolutions to that PR but not people with `Maintainers` role.

Additionally in the repo settings, there are no limitations about force pushing so it's really hard to tell what causes this issue.

We've already tried  Activating the `Restrict who can push to matching branches` and adding an exception for our dev team inside
to fix the issue but without success.

See the config for that protected branch below.

<img width="461" alt="Bildschirmfoto 2022-10-18 um 13 40 20" src="https://user-images.githubusercontent.com/1609148/196419875-b8478927-c2b6-4115-a566-8ed356fd2bda.png">

<img width="821" alt="image" src="https://user-images.githubusercontent.com/1609148/196419948-1b2503cd-c065-4468-9ed1-ab844784ea0b.png">

<img width="835" alt="Bildschirmfoto 2022-10-18 um 13 41 40" src="https://user-images.githubusercontent.com/1609148/196420167-48e66ebb-0743-4318-a7cf-f81e8fc119d9.png">

<img width="834" alt="Bildschirmfoto 2022-10-18 um 13 42 13" src="https://user-images.githubusercontent.com/1609148/196420176-abeaab02-7e33-4c2a-a2b5-66e3c61f2a85.png">

**Comment by ossanna16** [comment]
Hey there @vmalyi and thank you for asking a good question! I'm glad you found our community 🙂
